packages/app-project/stores/Project.js

```diff
@@ -0,0 +1,44 @@
+import { flow, types } from 'mobx-state-tree'
+import _ from 'lodash'
+import axios from 'axios'
+
+function transformResponse (response) {
+  return _.get(response, 'data.projects[0]')
+}
+
+function fetchProjectData (slug) {
+  return axios({
+    url: 'https://www.zooniverse.org/api/projects',
+    params: { slug },
+    headers: {
+      'Accept': 'application/vnd.api+json; version=1',
+      'Content-Type': 'application/json'
+    }
+  }).then(transformResponse)
+}
+
+const Project = types
+  .model('Project', {
+    isLoading: types.boolean,
+    data: types.optional(types.frozen, {})
+  })
+  .actions(self => ({
+    fetch: flow(function * fetch (slug) {
+      try {
+        self.isLoading = true
+        self.data = yield fetchProjectData(slug)
```

Comment:
I was thinking about adding to panoptes.js a helper request function that already had params defined to include the default linked resources we would want like background, avatar, etc. Basically a better version of this: https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/project/index.cjsx#L109-L154

Do you think that'd be helpful in the client?

Reply:
Definitely!